src/cts/test/CMakeLists.txt

```diff
@@ -7,29 +7,37 @@
 include("openroad")
 
 set(TEST_NAMES
+    simple_test_hier
+    array
```

Comment:
Any reason this isn't sorted?  Likewise lvt_lib

Reply:
order was kept from the record_tests list (but it would be trivial to sort them if you want)